README.md

```diff
@@ -1,64 +1,73 @@
-    An immutable, blockchain powered package registry for Deno.
+    A module registry for Deno, on the blockchain
  </p>
 </p>
 
-## About
+## What is nest.land?
 
-nest.land is a package registry that uses Arweave to upload your modules to the blockchain, keeping them secure and immutable.
+Nest.land combines Deno with the [Arweave Blockchain](https://www.arweave.org/). With us, you can publish your Deno modules to the permaweb, where they can never be deleted. This avoids a major pitfall for web-based module imports while allowing the developer to maximize on the potential of Deno's import design!
 
+## Repositories/Services
 
-## Roadmap
+nest.land is split into many services, each in their own respective repositories, they are as follows:
 
-See the [roadmap](/ROADMAP.md).
+- [nest.land](https://github.com/nestdotland/nest.land) - The landing repository for nest.land.
+  It houses the website source code.
+- [eggs](https://github.com/nestdotland/eggs) - nest.land's CLI
+- [yolk](https://github.com/nestdotland/yolk) - nest.land's API library
+- [website](https://github.com/nestdotland/website) - nest.land's website
+- [API](https://github.com/nestdotland/api) - nest.land's API
+- [twig](https://github.com/nestdotland/twig) - A mediator for nest.land's interactions with Arweave
+- [x](https://github.com/nestdotland/x) - The module gateway for files to be accessed.
 
+### Infrastructure Diagram
 
-## Project Diagram
-
-<img src="https://nest.land/nest-diagram.svg">
+The mentioned services are keystone to making nest.land a better service.
+This diagram is a visualisation of how these parts work together:
 
+![Diagram of nest.land infrastructure](./assets/infra_diagram.svg)
 
 ## Credits
 
 ### Authors
 
-- [t8](https://github.com/t8), Co-Founder, Frontend, & Project management
-- [zorbyte](https://github.com/zorbyte), Co-Founder & Advisor
-- [justablob](https://github.com/justablob), Backend
+- [t8](https://github.com/t8) - Co-Founder, Frontend, CLI & Triager
+- [zorbyte](https://github.com/zorbyte) - Co-Founder, Backend, Triager & Code Quality Control
+- [justablob](https://github.com/justablob) - Backend & CLI
 
 ### Core Team
 
-- [divy-work](https://github.com/divy-work)
-- [maximousblk](https://github.com/maximousblk)
-- [martondev](https://github.com/MartonDev)
-- [oganexon](https://github.com/oganexon)
+- [divy-work](https://github.com/divy-work) - Backend & CLI
+- [maximousblk](https://github.com/maximousblk) - Frontend & CLI
+- [martondev](https://github.com/MartonDev) - Frontend & CLI
+- [oganexon](https://github.com/oganexon) - CLI
 
 ### Contributors
 
-- [ebebbington](https://github.com/ebebbington)
-- [yg](https://github.com/yg)
-- [jletey](https://github.com/jletey)
-- [qu4k](https://github.com/Qu4k)
+- [ebebbington](https://github.com/ebebbington) - CLI
+- [yg](https://github.com/yg) - Triager
+- [jletey](https://github.com/jletey) - Backend
+- [qu4k](https://github.com/Qu4k) - Backend & Code Quality Control
```

Comment:
```suggestion
- [qu4k](https://github.com/Qu4k)
```

Reply:
I don't want roles for contributors, as they can change and will be a pain to manage at scale.